api/db-cdr/generate-cdr/README.md

```diff
@@ -0,0 +1,37 @@
+COUNT GENERATION
+
+This document describes what counts are generated for each of the data domains
+
```

Comment:
Please add bit about source concepts and count values. definition of count value and source count value like so: 
Most EHR data in OMOP has two concepts associated with it , the source concept and the standard concept that source maps to.  So we calculate two counts for each concept 
countValue is the number of distinct patients that had that concept or some concept that mapped to it. 
sourceCountValue is the number of distinct patients that had that concept as real ehr data.
In the case of source concepts that always map to a standard concept, the countValue would be zero. In that case we set countValue to the sourceCountValue as it is more userfriendly in apis and front end development. 
For standard concepts , the count value will be much more than the sourceCountValue in general. 

That's a start :)

Reply:
Addressed.